packages/strapi-connector-bookshelf/lib/build-database-schema.js

```diff
@@ -278,21 +279,21 @@ const createOrUpdateTable = async ({ table, attributes, definition, ORM, model }
     return;
   }
 
-  const columns = Object.keys(attributes);
+  const attributesNames = Object.keys(attributes);
 
   // Fetch existing column
   const columnsExist = await Promise.all(
-    columns.map(attribute => ORM.knex.schema.hasColumn(table, attribute))
+    attributesNames.map(attribute => ORM.knex.schema.hasColumn(table, attribute))
   );
 
   const columnsToAdd = {};
 
   // Get columns to add
   columnsExist.forEach((columnExist, index) => {
-    const attribute = attributes[columns[index]];
+    const attribute = attributes[attributesNames[index]];
 
     if (!columnExist) {
-      columnsToAdd[columns[index]] = attribute;
+      columnsToAdd[attributesNames[index]] = attribute;
```

Comment:
I think we should refactor that bit of code to get the columnsTo add either using an async for loop or make it more functional. here are 2 examples https://repl.it/@alexandrebodin/WelllitFuzzyQuadrants#index.js